source/jormungandr/jormungandr/street_network/taxi.py

```diff
@@ -133,10 +133,10 @@ def _add_additional_section_after_first_section_taxi(self, journey, pt_object, a
         # Because Jormun does not do it afterwards
         journey.sections[-1].destination.CopyFrom(pt_object)
 
-        self._update_journey(journey, additional_section)
+        self._update_journey(journey, additional_section, 1)
 
-    def _add_additional_section_before_first_section_taxi(self, journey, pt_object, additional_time):
-        logging.getLogger(__name__).info("Creating additional section_before_first_section_taxi")
+    def _add_additional_section_before_last_section_taxi(self, journey, pt_object, additional_time):
```

Comment:
Are you sure about this being `last` ? It looks like we create a new section at index `0` :thinking:

Reply:
When you use last_section_mode[]=taxi, you create an additional section before this one